Tools/scripts/pathfix.py

```diff
@@ -64,6 +67,11 @@ def main():
             create_backup = False
         if o == '-k':
             keep_flags = True
+        if o == '-a':
+            add_flags = a.encode()
+            if ' ' in a:
```

Comment:
I suggest to check for b' ' in add_flags (bytes) instead, since add_flags is used later, not a.